.editorconfig

```diff
@@ -114,7 +114,7 @@ dotnet_style_prefer_conditional_expression_over_return = true:silent
 dotnet_style_prefer_conditional_expression_over_assignment = true:silent
 dotnet_style_prefer_auto_properties = true:suggestion
 csharp_prefer_simple_default_expression = true:suggestion
-dotnet_style_prefer_compound_assignment = false:silent
+dotnet_style_prefer_compound_assignment = true:suggestion
```

Comment:
Please remove. It is already in .globalconfig file as warning.

Reply:
@iSazonov, done